jcl/src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -259,7 +256,8 @@ final Reference<? extends T> remove0() throws IllegalArgumentException, Interrup
 }
 
 final boolean enqueue0(Reference reference) {
-	return enqueue(reference);
+	enqueue(reference);
+	return true;
 }
```

Comment:
`true` isn't necessarily the right answer. I suggest this method just be removed: it would only be used by `NativeReferenceQueue` if that class were not overridden for OpenJ9.